webui/react/src/utils/table.ts

```diff
@@ -0,0 +1,17 @@
+import { GridCell, GridCellKind } from '@glideapps/glide-data-grid';
+
+export const EMPTY_CELL: GridCell = {
+  allowOverlay: false,
+  data: '-',
+  displayData: '-',
+  kind: GridCellKind.Text,
+} as const;
+
+export const handleEmptyCell = <T>(
+  param: T | undefined,
+  cell: (datum: T) => GridCell,
+  onlyUndefined = true,
+): GridCell => {
+  if ((onlyUndefined && param === undefined) || (!onlyUndefined && !param)) return EMPTY_CELL;
```

Comment:
this works with the current columns, but if we wanna show `EMPTY_CELL` for special cases, this wouldnt work. For example, show `-` when data is `EMPTY` or `-1` etc. i feel like its safer not to use `handleEmptyCell`. What do you think?

Reply:
I think that seems unlikely, but in that case we could either update `handleEmptyCell` to deal with it or handle the special case when we call it, like `handleEmptyCell(variable !== 'EMPTY' ? variable : undefined, ...`

Reply:
got it